packages/widgetbook_addon/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 3.0.0-beta.0
+
+- Initial version.
```

Comment:
Need to improve this later, but might be covered by melos changelog generation anyways